app/src/main/java/ro/code4/monitorizarevot/ui/login/LoginActivity.kt

```diff
@@ -11,16 +14,24 @@ import ro.code4.monitorizarevot.helper.startActivityWithoutTrace
 import ro.code4.monitorizarevot.ui.base.BaseActivity
 import ro.code4.monitorizarevot.ui.branch.BranchActivity
 import ro.code4.monitorizarevot.ui.onboarding.OnboardingActivity
+import ro.code4.monitorizarevot.widget.ProgressDialogFragment
 
 class LoginActivity : BaseActivity<LoginViewModel>() {
 
+    private val progressDialog: ProgressDialogFragment by lazy { ProgressDialogFragment() }
```

Comment:
The `dialog` should be dismissed if the activity is dismissed/closed (e.g: the user dismisses the app while a dialog is shown and a `java.lang.IllegalArgumentException: View not attached to window manager` exception is thrown)